### PR TITLE
Add integrity hashes for security.

### PR DESCRIPTION
**Pull request by TexaCaliCoder**
## Add integrity hashes for security: 
- added `webpack-subresource-plugin` package
- updated `.prettierrc` to keep trailing commas in place.
- updated version number.

We are adding this package to prevent MITM (man in the middle) proxies from intercepting our `.js` resources and injecting malicious scripts.

### CODE REVIEW

File: packages/react-scripts/package.json [hunk 1]
```diff
@@ -87,6 +87,7 @@
     "webpack-bundle-analyzer": "3.9.0",
     "webpack-dev-server": "3.11.1",
     "webpack-manifest-plugin": "2.2.0",
+    "webpack-subresource-integrity": "1.5.2",
```
Comment (m2mathew) [inline packages/react-scripts/package.json:90]: The latest [version](https://www.npmjs.com/package/webpack-subresource-integrity) of this package is `5.0.0`. It's the next version after this `1.5.2`, for some reason. 🤔 

Is there a reason that we are targeting the older package? Maybe the `5.0.0` version doesn't work with `react-scripts`'s version of webpack?

**Review by m2mathew — COMMENTED** [review]
I have the one question. Also, I need to run it...I'll do that next.

File: packages/react-scripts/package.json (hunk 1 above)
Comment (TexaCaliCoder) [reply packages/react-scripts/package.json:90]: The reason we are using 1.5.2 is because we are using webpack 4.

**Review by m2mathew — APPROVED** [review]
👍

**Pull request closed without merging**